docs/framework/configure-apps/file-schema/runtime/gcallowverylargeobjects-element.md

```diff
@@ -51,7 +51,7 @@ On 64-bit platforms, enables arrays that are greater than 2 gigabytes (GB) in to
 ## Remarks  
  Using this element in your application configuration file enables arrays that are larger than 2 GB in size, but does not change other limits on object size or array size:  
   
-- The maximum number of elements in an array is <xref:System.UInt32.MaxValue?displayProperty=nameWithType>.  
+- The maximum number of elements in an array is <xref:System.Int32.MaxValue?displayProperty=nameWithType>.  
```

Comment:
why this change? it's declared as an unsigned int in the code...

Reply:
I've reverted the change, @Maoni0.